src/store/index.ts

```diff
@@ -394,6 +404,41 @@ const resolvers: Resolvers = {
                 did: user?.did!,
                 loggedIn: false,
             }
+        },
+        acceptJob: async (_root, {input: {user,trackable}}:MutationAcceptJobArgs, {cache,communityPromise}: TrackerContext): Promise<AcceptJobPayload|undefined> => {
+            if (!appUser.userPromise) {
+                return undefined
+            }
+            let loggedinUser = await appUser.userPromise
+            if (!loggedinUser || (loggedinUser.did !== user)) {
+                return undefined
+            }
+            await loggedinUser.load()
+    
+            // first we update the trackable
+            // TODO: DRY This up from the addUpdate resolver
+
+            // TODO: this needs to work on ChainTrees that are not owned by
+            // the user doing the owning - so the below won't quite work yet.
+
+            // let timestamp = (new Date()).toISOString()
+
+            // const trackableTree = await Tupelo.getLatest(trackable)
+            // trackableTree.key = loggedinUser.tree.key
+
+            // let update:TrackableUpdate = {
+            //     did: `${(await trackableTree.id())}-${timestamp}`,
+            //     timestamp: timestamp,
```

Comment:
timestamp seems pretty valuable from a ux perspective still to me, especially from the "admin" or "donator oversight" sides?

Reply:
yeah for sure... this code needs to be uncommented... just don't have permissions right now.